data/json/recipes/food/brewing.json

```diff
@@ -228,15 +228,9 @@
     "tools": [ [ [ "surface_heat", 6, "LIST" ] ] ],
     "components": [
       [ [ "water", 10 ], [ "water_clean", 10 ] ],
-      [ [ "sugar", 2 ] ],
+      [ [ "sugar", 2 ], [ "artificial_sweetener", 2 ] ],
```

Comment:
Isn't this sugar supposed to be a substrate for the yeast 🤔

Reply:
Yes, you're correct on both points.

Reply:
I removed it